mater/lib/src/stores/filestore.rs

```diff
@@ -173,6 +274,92 @@ mod test {
         assert_buffer_eq!(&expected, &result);
     }
 
+    #[tokio::test]
+    async fn test_filestore_unixfs_dag_structure() {
```

Comment:
Can we add a test which checks whether this is somehow similar to the reference implementation?

Reply:
Yes please, we need to CAR-ify a large-ish file (like glenda) and commit it to do a byte-check